voting-webapp/experiment.yaml

```diff
@@ -94,23 +93,18 @@ spec:
                   memory: "{{ .Values.db_memory }}Mi"
                 requests:
                   memory: "{{ .Values.db_memory }}Mi"
-  template: # trial
+  trialTemplate:
     spec:
-      template: # job
+      jobTemplate:
         spec:
-          template: # pod
+          template:
             spec:
               containers:
               - name: locust
                 image: grubykarol/locust:0.12.0-python3.7-alpine3.10
                 command: ["/usr/local/bin/locust"]
-                args: ["--locustfile=/locust-tasks/locustfile.py",
-                       "--no-web",
-                       "--host=http://voting-service",
-                       "--clients=200",
-                       "--hatch-rate=20",
-                       "--run-time=300",
-                       "--exit-code-on-error=0"]
+                args: ["--locustfile=/locust-tasks/locustfile.py", "--no-web", "--host=http://voting-service",
+                  "--clients=200", "--hatch-rate=20", "--run-time=300", "--exit-code-on-error=0"]
```

Comment:
would we be able to change this from a json style array/list to yaml to improve readability?
```
args:
- --locustfile=/locust-tasks/locustfile.py
- --no-web
- ...
```